scripts/badger/bridge_out_arbitrum_fees/wd_on_mainnet.py

```diff
@@ -0,0 +1,32 @@
+import json
+import os
+from brownie import interface
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+
+def main(msig="techops_multisig", token_bridge="CRV", batchNum=0, index=0):
+    safe = GreatApeSafe(registry.eth.badger_wallets[msig])
```

Comment:
ah do i understand correctly techops is withdrawing for treasury ops?

Reply:
just a msig as default from last test, but yeah makes more sense trops 🤝